test/mocha/cli/cli.spec.js

```diff
@@ -3,6 +3,33 @@ const { delayPromise } = require('../../helpers/utils');
 const { execCommand } = require('cli-testlab');
 
 describe('cli', () => {
+  describe('columns', function () {
+    this.timeout(5000);
+    it('manages columns successfully', function () {
+      return execCommand(`node bin/forever columns reset`,
```

Comment:
this is testing whether `forever columns reset` works, it was throwing `TypeError: forever.config.saveSync is not a function` originally